x-pack/plugins/transform/public/app/app_dependencies.tsx

```diff
@@ -7,25 +7,39 @@
 import React, { createContext, useContext, ReactNode } from 'react';
 import { HashRouter } from 'react-router-dom';
 
+import { CoreSetup, CoreStart } from 'src/core/public';
+import { DataPublicPluginStart } from 'src/plugins/data/public';
+
 import { API_BASE_PATH } from '../../common/constants';
 
 import { setDependencyCache } from '../shared_imports';
-import { AppDependencies } from '../shim';
 
 import { AuthorizationProvider } from './lib/authorization';
 
+export interface AppDependencies {
+  chrome: CoreStart['chrome'];
+  data: DataPublicPluginStart;
+  docLinks: CoreStart['docLinks'];
+  http: CoreSetup['http'];
+  i18n: CoreStart['i18n'];
+  notifications: CoreStart['notifications'];
+  uiSettings: CoreStart['uiSettings'];
+  savedObjects: CoreStart['savedObjects'];
+  overlays: CoreStart['overlays'];
+}
+
 let DependenciesContext: React.Context<AppDependencies>;
 
 const setAppDependencies = (deps: AppDependencies) => {
   const legacyBasePath = {
```

Comment:
this doesn't appear to be being used other than to set `basePath` in the dependency cache, which could be just `deps.http.basePath`